x-pack/plugins/security_solution/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -56,6 +65,9 @@ export const PolicyList = memo(() => {
     perPage: pagination.pageSize,
   });
 
+  const { data: outdatedManifestsCount, isLoading: outdatedManifestsCountLoading } =
```

Comment:
Can we get the `outdatedManifestsCount` directly with:

```typescript
const { data: { outdatedManifestsCount }, isLoading:  isOutdatedManifestsCountLoading} = ...
```

So we avoid this below:`outdatedManifestsCount.outdatedManifestsCount`

If not, can we just change the const name as `outdatedManifestsCountResponse` or something different?

Reply:
Went with `outdatedManifestsCountResponse`.